tests/unit/test_column_noise.py

```diff
@@ -125,9 +119,149 @@ def test_miswrite_zipcodes():
     pass
 
 
-@pytest.mark.skip(reason="TODO")
-def test_miswrite_ages():
-    pass
+def test_miswrite_ages_default_config(dummy_dataset):
+    """Test that miswritten ages are appropriately handled, including
+    no perturbation probabilities defaults to uniform distribution,
+    perturbation probabilities"""
+    config = get_configuration()["decennial_census"]["age"]["age_miswriting"]
+    data = dummy_dataset.rename(columns={"numbers": "age"})["age"]
+
+    # Convert to realistic age
+    maximum_age = 120
+    data = data.apply(pd.to_numeric, args=("coerce",))
+    data = data / data.max() * (maximum_age + 1)
+    data[data.isna()] = -1  # temp nan
+    data = data.astype(int).astype(str)
+    data[data == "-1"] = ""
```

Comment:
What is this for?

Reply:
We temporarily store missingness as -1 so that we can convert to int and then str. It's just a hamfisted way of getting the float data to be age-like (string integers with "" for missingingess)